analyses/molecular-subtyping-embryonal/01-samples-to-subset.Rmd

```diff
@@ -77,14 +91,37 @@ ttyh1_fusions <- fusion_summary_df %>%
 Which samples have these fusions?
 
 ```{r}
-fusion_biospecimens <- ttyh1_fusions %>%
+ttyh1_fusion_biospecimens <- ttyh1_fusions %>%
   # add a column that counts how many TTYH1 fusions are present
   mutate(fusion_count = rowSums(select(., starts_with("TTYH1")))) %>%
   # if there's any TTYH1 fusion - pull out the biospecimen ID
   filter(fusion_count > 0) %>%
   pull(Kids_First_Biospecimen_ID)
 ```
 
+### _MN1_ fusions
```

Comment:
@jharenza this was put in place to specifically make sure we captured the EPN specimen you mentioned in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/785#issuecomment-695015488, but adding this in nets us an LGG sample:

| `Kids_First_Participant_ID` | `sample_id` | `Kids_First_Biospecimen_ID_DNA` | `Kids_First_Biospecimen_ID_RNA` | new `molecular_subtype` | old `molecular_subtype` | status change |
|-----------------|------------|--------------------|--------------------|-----------------|----------------|-------|
| PT_E3ADF4ZB |	7316-2664 | BS_C1RRDNF5 | BS_ZVZDDW2G	 | CNS Embryonal, NOS | LGG, BRAF wildtype | added (on the basis of `MN1` fusions)|

Per https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/788#issuecomment-698598761 - thoughts? It is `MN1--PATZ1` if that's helpful information.

Reply:
Quickly looked this up and found [this paper](https://onlinelibrary.wiley.com/doi/epdf/10.1111/nan.12626) which suggests this is an entity **separate** of the HGNET-MN1 tumors, so I would keep that sample/fusion out of this analysis:

>In conclusion, MN1‐PATZ1 fusions may define a novel histological and immunohistochemical polyphenotypic malignant paediatric brain tumour, not showing epigenetic similarities with CNS HGNET‐MN1, but closely related to PATZ1‐sarcomas.